src/bin/krimedc.rs

```diff
@@ -449,6 +459,11 @@ impl TryFrom<Config> for ServerState {
 struct PrincipalRecord {
     service: bool,
     base_key: DerivedKey,
+    // TODO Keep track of last request time.
```

Comment:
I suspect there are information leaks about this, so I think we should flip this to "what does setting the last request time do for us? How is it a benefit?"

Reply:
No real benefit, and you are right about the leaks. I have modified the PR to remove most of it except adding one element to the sequence.